easybuild/easyconfigs/e/EGSnrc/EGSnrc-2019a.eb

```diff
@@ -0,0 +1,94 @@
+easyblock = 'Tarball'
+name = 'EGSnrc'
+version = '2019a'
+
+homepage = 'http://nrc-cnrc.github.io/EGSnrc/'
+description = """
+EGSnrc is a software toolkit to perform Monte Carlo simulation of ionizing radiation transport through matter.\
+It models the propagation of photons, electrons and positrons with kinetic energies\
+between 1 keV and 10 GeV, in homogeneous materials."""
+
+source_urls = ['https://github.com/nrc-cnrc/EGSnrc/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = ['log-dir_finanlize.patch']
+checksums = [
+    'd6f9198aa2d27645134cb5be477165547df6a66b2c0eed11e73c7d118ea312a8',  # v2019a.tar.gz
+    'd54989aead38e7112d327a7ff299587c99c9d2b5c5df981322ebf1d354df34f5',  # log-dir_finanlize.patch
+]
+
+toolchain = {'name': 'system', 'version': ''}
+
+dependencies = [
+     ('GCC', '6.4.0-2.28'),
```

Comment:
But if you need GCC as a dependency you should not use the SYSTEM toolchain.
Use toolchain = {'name': 'GCC', 'version': '6.4.0-2.28'} instead.

Or preferably use a newer toolchain if possible.